docs/create/from-source/index-from-source.mdx

```diff
@@ -43,9 +54,12 @@ redis> get foo
 ```
 
 Please note: By default, Redis has 0-15 indexes for databases, you can change that number databases NUMBER in redis.conf.
```

Comment:
```suggestion
```

Removing as we discourage use of this and the `SELECT` command and Redis Enterprise doesn'thave it.